certified-connectors/Maximizer/apiDefinition.swagger.json

```diff
@@ -399,7 +399,279 @@
                         "description": "AbEntry"
                       },
                       "Opportunity": {
-                        "type": "string",

```

Comment:
@davidchanmaximizer Typically the below errors are breaking changes, but since this is a new connector we can ignore them. 
```
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/FindLead/responses/200/schema/Values/items/Opportunity/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/TriggerNewLead/responses/200/schema/Values/items/AssignedTo/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/TriggerNewLead/responses/200/schema/Values/items/AbEntry/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/TriggerNewLead/responses/200/schema/Values/items/Opportunity/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/TriggerUpdateLead/responses/200/schema/Values/items/AssignedTo/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/TriggerUpdateLead/responses/200/schema/Values/items/AbEntry/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/TriggerUpdateLead/responses/200/schema/Values/items/Opportunity/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/LeadsCreate/responses/200/schema/Values/items/CreatedEntry/AbEntry/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/LeadsCreate/responses/200/schema/Values/items/CreatedEntry/Opportunity/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/LeadsUpdate/responses/200/schema/Values/items/CreatedEntry/items/AbEntry/type Source: NewSwagger
##[error]The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. ValuePath: operation/LeadsUpdate/responses/200/schema/Values/items/CreatedEntry/items/Opportunity/type Source: NewSwagger
```